### ISSUE TITLE
[$500] Desktop - Login - Animation (only the bottom part) flashes during app launch

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#35556]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**  1.4.34-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://expensify.testrail.io/index.php?/tests/view/4257388
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

## Action Performed:
1. Open the app
2. Reload the app several times to see the animation

## Expected Result:
The animation shouldn't flash during app launch

## Actual Result:
Animation (only the bottom part) flashes during app launch

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/20a97f94-99f4-4e2f-a9d4-ee9917492e4b

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01ba62285808441524</li>
        <li>Upwork Job ID: 1753059301658746880</li>
        <li>Last Price Increase: 2024-02-01</li>
    </ul>
</details>

**Comment by lanitochka17** [comment]
We think that this bug might be related to #wave9-collect-signup
CC @MitchExpensify

**Comment by Atta-ur-rehman01** [comment]
I came to know that you're dealing with an animation glitch in your app launch. Having reviewed the details, I'm confident in my ability to address this issue. With experience in app development and a keen eye for seamless user experiences, I believe I can contribute to fixing the flashing animation problem. Let's discuss how I can assist you in enhancing the user experience on your app.

**Comment by Atta-ur-rehman01** [comment]
Contributor details
Your Expensify account email: bizzclan2@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01066fdf37fb4931e2

**Comment by ikevin127** [comment]
# Proposal

### Please re-state the problem that we are trying to solve in this issue

The `Hands` login lottie animation flashes during app launch on MacOS: Desktop.

### What is the root cause of that problem?

As discussed in [this issue](https://github.com/lottie-react-native/lottie-react-native/issues/1170), because by default the `renderMode` is "AUTOMATIC" which is the new default, on certain platforms this can have performance issues like this issue's glitch / flash.

From the mentioned discussion:
> By default, It has an automatic variant that picks based on environment where to run it on ("AUTOMATIC") and that is the new default. "SOFTWARE" is the old behaviour from V5 that handles all the compute on the CPU side, And "HARDWARE" is explicitly using the newer mode with GPU Utilization.

### What changes do you think we should make in order to solve the problem?

**TLDR**: Change the `renderMode` to "HARDWARE" which relies on GPU to render the animation.

To do this we have to set the `renderMode='HARDWARE'` prop in the `SignInHeroImage` component.
In order to avoid messing up the default renderMode on other platforms we can use a library that targets only desktop, maintaining the default renderMode on the other platforms.

### Videos

<details>
<summary>MacOS: Desktop</summary>

| Before | After |
| ------------- | ------------- |
| <video src="https://github.com/Expensify/App/assets/56457735/a793eaa3-d06d-41be-a989-38c3d1f2a4ea"> | <video src="https://github.com/Expensify/App/assets/56457735/d0395829-fd3a-4ef5-8200-37c0f2fb048e"> |

</details>

**Comment by s77rt** [comment]
@Atta-ur-rehman01 @ZhenjaHorbach Can you please follow the proposal template

**Comment by s77rt** [comment]
@ZhenjaHorbach Thanks for the proposal. Your RCA makes sense. I guess this needs to be fixed by the design team.


PS: I didn't notice that you updated your proposal. Next time please tag me on proposal updates.

**Comment by s77rt** [comment]
@Expensify/design The Hands animation first frames are "hiding" the content, this makes the animation looks buggy at the very first launch. Can you please look into this?

**Comment by s77rt** [comment]
Indeed. Closing makes sense. cc @sonialiap

**Comment by MitchExpensify** [comment]
Closing as Sonia is OOO and I agree this isn't really wave 9 worthy
